pipelines/e2e/Dockerfile.setup

```diff
@@ -25,7 +25,7 @@ RUN wget -q https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_V
     wget -q "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64" && \
     mv yq_linux_amd64 /usr/local/bin/yq && \
     chmod +x /usr/local/bin/yq && \
-    apk add openssl openssh-client ca-certificates git rsync bash curl jq python3 py3-pip && \
+    apk add openssl openssh-client ca-certificates git rsync bash curl jq python3 py3-pip gcc python3-dev libc-dev && \
```

Comment:
What's purpose to have gcc and libc here?

Reply:
To satisfy package installation in `requirements.txt`.

Reply:
Do we know which package requires those?

Reply:
```
# pip install directio==1.2
Collecting directio==1.2
  Using cached directio-1.2.tar.gz (3.1 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: directio
  Building wheel for directio (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [9 lines of output]
      Warning: 'classifiers' should be a list, got type 'filter'
      running bdist_wheel
      running build
      running build_ext
      building 'directio' extension
      creating build
      creating build/temp.linux-x86_64-3.10
      gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fomit-frame-pointer -g -fomit-frame-pointer -g -fomit-frame-pointer -g -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/include/python3.10 -c directiomodule.c -o build/temp.linux-x86_64-3.10/directiomodule.o
      error: command 'gcc' failed: No such file or directory
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for directio
  Running setup.py clean for directio
Failed to build directio
Installing collected packages: directio
  Running setup.py install for directio ... error
  error: subprocess-exited-with-error
  
  × Running setup.py install for directio did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      Warning: 'classifiers' should be a list, got type 'filter'
      running install
      /usr/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_ext
      building 'directio' extension
      creating build
      creating build/temp.linux-x86_64-3.10
      gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fomit-frame-pointer -g -fomit-frame-pointer -g -fomit-frame-pointer -g -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/include/python3.10 -c directiomodule.c -o build/temp.linux-x86_64-3.10/directiomodule.o
      error: command 'gcc' failed: No such file or directory
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> directio

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```